source/common/sagemath/library.sage

```diff
@@ -222,13 +238,15 @@ class TBIL:
         return P
 
     @staticmethod
-    def typeset_angle(theta):
-        angle_string=f"${latex(theta)}$"
+    def typeset_angle(theta,latex_delimiters=False):
+        angle_string=f"{latex(theta)}"
         if type(theta) is sage.symbolic.expression.Expression and theta.is_rational_expression() and denominator(theta)!=1:
             if numerator(theta)<0:
-                angle_string = f"$-\\dfrac{{{latex(-1*numerator(theta))}}}{{{denominator(theta)}}}$"
+                angle_string = f"-\\dfrac{{{latex(-1*numerator(theta))}}}{{{denominator(theta)}}}"
             else:
-                angle_string = f"$\\dfrac{{{latex(numerator(theta))}}}{{{denominator(theta)}}}$"
+                angle_string = f"\\dfrac{{{latex(numerator(theta))}}}{{{denominator(theta)}}}"
+        if latex_delimiters:
+            angle_string="$"+angle_string+"$"
         return angle_string
```

Comment:
Changed this to not include LaTeX style delimiters (which are expected in labels of certain plotting functions in Sage) by default.